pkgs/data/themes/catppuccin-kde/color.sh

```diff
@@ -0,0 +1,61 @@
+echo $FLAVOUR $ACCENT $WINDECSTYLE
+
+FLAVOURNAME=${FLAVOUR^}
+ACCENTNAME=${ACCENT^}
+WINDECSTYLENAME=${WINDECSTYLE^}
+
+if [[ $FLAVOURNAME == "Mocha" ]]; then
+    FLAVOUR="1";
+elif [[ $FLAVOURNAME == "Macchiato" ]]; then
+    FLAVOUR="2";
+elif [[ $FLAVOURNAME == "Frappe" ]]; then
+    FLAVOUR="3";
+elif [[ $FLAVOURNAME == "Latte" ]]; then
+    FLAVOUR="4";
+fi
+if [[ $FLAVOURNAME == "Mocha" ]]; then
```

Comment:
Can you please explain the reason for this repeating code block, is this supposed to be here or was it a mistake

Reply:
Definitely just a mistake. I did most of this using macros in helix, so I didn't realize I had gone over the same block twice.